frontend/workflows/projectCatalog/src/catalog/index.tsx

```diff
@@ -135,17 +135,12 @@ const Catalog: React.FC<WorkflowProps> = ({ heading }) => {
         <div style={{ margin: "16px" }}>
           <Form noValidate onSubmit={handleSubmit(triggerProjectAdd)}>
             <TextField
-              placeholder="Search"
+              label="Search"
+              placeholder="Project Name"
               value={state.search}
               onChange={handleChanges}
               autocompleteCallback={v => autoComplete(v)}
-              endAdornment={
-                state.isSearching ? (
-                  <CircularProgress size="24px" />
-                ) : (
-                  <SearchIcon onClick={triggerProjectAdd} />
```

Comment:
Why remove the `onClick` prop?

Reply:
It throws an extremely annoying error in the UI:

```
Warning: Failed prop type: MUI: You are providing an onClick event listener to a child of a button element.
Prefer applying it to the IconButton directly.
This guarantees that the whole <button> will be responsive to click events.
```

Since we already handle this in the submit it seemed unnecessary.